lib/screens/transfer/send/send_enter_data/send_enter_data_screen.dart

```diff
@@ -63,6 +63,7 @@ class SendEnterDataScreen extends StatelessWidget {
           } else if (command is ShowTransferSuccess) {
             if (command.shouldShowInAppReview) {
               InAppReview.instance.requestReview();
+              settingsStorage.saveDateSinceLastAsked(DateTime.now().millisecondsSinceEpoch);
```

Comment:
same here

Reply:
I like it here right after it ask the user, in case the app crashes, but can be move to the mapper or bloc right after the _shouldShowInAppReview logic